backend/endpoints/rooms.py

```diff
@@ -75,6 +78,75 @@ async def create_room(
         detail="unable to create room",
     )
 
+@router.patch("/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    response_model=ResponseModel, 
+    status_code=status.HTTP_200_OK,
+    responses={
+        200: {"detail": {"room": "room"}},
+        404: {"detail": "room or member not found"},
+        424: {"detail": "member removal unsuccessful"},
+    },)
+async def remove_member(org_id: str, member_id: str, room_id: str, admin_id: str):
+    """Removes a member from a room either when removed by an admin or member leaves the room.
+
+    Fetches the room which the member is removed from from the database collection
+    Pops the member being removed from the room's members dict
+    Updates the database collection with the new room
+    Returns the room dict if member was removed successfully
+
+    Args:
+        org_id (str): A unique identifier of an organisation
+        member_id (str): A unique identifier of the member being removed from the room
+        room_id (str): A unique identifier of the room a member is being removed from
+        admin_id (str): A unique identifier of the member removing another member
+
+    Returns:
+        HTTP_200_OK (member removed from room): {room}
+    Raises
+        HTTP_404_NOT_FOUND: room or member not found
+        HTTP_403_FORBIDDEN: not authorized to remove room  member
+        HTTP_424_FAILED_DEPENDENCY: member removal unsuccessful
+    """
+    room_obj = await get_room(org_id, room_id)
+
+    if not room_obj:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="room does not exist",
+        )
+    
+    if room_obj["room_type"] != RoomType.CHANNEL:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="cannot remove member from DM rooms",
+        )
+    
+    if member_id not in room_obj["room_members"]:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="user not a member of the room",
+        )
+    
+    if admin_id == None:
```

Comment:
Since `None` is a singleton, it's preferable to use the identity keyword `is` to check if a variable is None